app/assets/styles/utilities/main.css

```
@@ -0,0 +1,356 @@
+}
+._pb-10 {
+  padding-bottom: 10rem !important;
+}
+
+/* line-clamp */
+/* @see https://twitter.com/tak_dcxi/status/1766517419466793102?s=12&t=0Bs_ltBYiO3nhiiL9YZAEw */
+
+._line-clamp-1 {
+  display: -webkit-box;
+  text-overflow: ellipsis;
+  -webkit-box-orient: block-axis;
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "-webkit-box-orient" (property-no-deprecated)

(property-no-deprecated)